src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/rules/draggableList/sortableItem.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {DraggableSyntheticListeners} from '@dnd-kit/core';
+import {useSortable} from '@dnd-kit/sortable';
+import {Transform} from '@dnd-kit/utilities';
+
+type Props = {
+  id: string;
+  renderItem(args: {
+    value: React.ReactNode;
+    listeners?: DraggableSyntheticListeners;
+    transform?: Transform | null;
+    transition?: string;
+    style?: React.CSSProperties;
+    forwardRef?: React.Ref<HTMLElement>;
+  }): React.ReactElement | null;
```

Comment:
Should the type for `renderItem()` be exported? I could see it being useful for when you need types for sortable children.

Reply:
yes, but I think I prefer to do that when creating the sortable List component 😉